hdt-lib/src/hdt/BasicHDT.cpp

```diff
@@ -656,7 +657,7 @@ void BasicHDT::loadFromHDT(std::istream & input, ProgressListener *listener)
 	controlInformation.load(input);
 	std::string hdtFormat = controlInformation.getFormat();
 	if(hdtFormat!=HDTVocabulary::HDT_CONTAINER) {
```

Comment:
Isn't there any built-in regex support we can rely on (probably not)? Or `scanf`?